pkgs/by-name/me/mealie/package.nix

```diff
@@ -92,9 +92,17 @@ pythonpkgs.buildPythonPackage rec {
     uvicorn
   ];
 
-  postPatch = ''
+  patchPhase = ''
```

Comment:
Keep it as a `postPatch` please, that way someone can still rely on `patches` working when overriding the package.